Formula/libzip.rb

```diff
@@ -20,7 +20,10 @@ class Libzip < Formula
 
   depends_on "cmake" => :build
 
+  uses_from_macos "zip" => :test
   uses_from_macos "bzip2"
+  uses_from_macos "openssl@1.1"
+  uses_from_macos "xz"
   uses_from_macos "zlib"
 
   conflicts_with "libtcod", "minizip2",
```

Comment:
```suggestion
  conflicts_with "libtcod", "minizip-ng",
```
Formula was renamed, so would be good to fix it in one go. Reason below also needs fixing.